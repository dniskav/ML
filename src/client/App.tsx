import React from 'react';
import styled from 'styled-components';
import Header from './components/Header';
import SearchResults from './components/SearchResults/index';
import BreadCrumbs from './components/BreadCrumbs/index';
import './styles/common.scss';
import { useSelector } from 'react-redux';
import { getResults } from './redux/selectors/index';
import Router from './components/Router/'

const Content = styled.div`
  display:flex;
  margin: 0 auto;
  flex-direction: column;
  width: 994px;
`;

const Container = styled.div`
  background: #eee;
  box-sizing: border-box;
  min-height: 230px;
`;

const App: React.FC = () => {
  const res = useSelector(getResults);
  return (
    <Container>
      <Header />
      <Content>
        <BreadCrumbs />
        <Router />
        <SearchResults results={res}/>
      </Content>
    </Container>
  );
}

export default App;
