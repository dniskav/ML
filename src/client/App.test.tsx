import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import App from './App';

describe('<App />', () => {
  
  it("render success", () => {
    const wrapper: ShallowWrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
