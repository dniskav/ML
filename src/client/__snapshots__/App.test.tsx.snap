// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> render success 1`] = `ShallowWrapper {}`;
